MNE/utils/mp/adaptivemp.cpp

```diff
@@ -209,7 +209,7 @@ QList<QList<GaborAtom> > AdaptiveMp::matching_pursuit(MatrixXd signal, qint32 ma
         k = 0;
         s = sample_count;
         p = floor(sample_count / 2);
-        j = floor(log2(sample_count));//log(sample_count) / log(2));
+        j = floor(log10(sample_count));//log(sample_count) / log(2));
```

Comment:
>> @@ -209,7 +209,7 @@ QList<QList<GaborAtom> >
>> AdaptiveMp::matching_pursuit(MatrixXd signal, qint32 ma
>>          k = 0;
>>          s = sample_count;
>>          p = floor(sample_count / 2);
>> -        j = floor(log2(sample_count));//log(sample_count) / log(2));
>> +        j = floor(log10(sample_count));//log(sample_count) / log(2));
>
> Is there a specific reason to change this line?
No. Just for passing compiler.

> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mne-tools/mne-cpp/pull/170/files#r26209195



The information in this e-mail is intended only for the person to whom it is
addressed. If you believe this e-mail was sent to you in error and the e-mail
contains patient information, please contact the Partners Compliance HelpLine at
http://www.partners.org/complianceline . If the e-mail was sent to you in error
but does not contain patient information, please contact the sender and properly
dispose of the e-mail.